### DISCUSSION TITLE
zigbee2mqtt groups are messed up

### DISCUSSION THREAD

**Discussion by EcarGaming** [Koenkk/zigbee2mqtt#18604] [General]
Hello everyone,
I've encountered an issue with my Home Assistant setup involving zigbee2mqtt and the Mosquitto broker.

My groups used to work really well previously, but now there are errors in them.
When I turn my "vitus_lights" group on or off using an "ON" or "OFF" command, a completely different group turns on and off instead, namely the "garden_lights" group. When I try to turn the "garden_lights" group on or off, nothing happens.

I've tried deleting all groups using MQTT Explorer and attempted to delete the groups in the zigbee2mqtt web interface in Home Assistant, followed by recreating them. However, the result remains the same.

Is there a solution that doesn't involve deleting the entire zigbee2mqtt setup and avoids having to re-pair all of my devices?

**Comment by Yrarara** [comment]
Hi! i want to help

**Comment by EcarGaming** [comment]
Thx. Now I have make a new installation. It still do the same. When i send a turn off to The group “vitus_lys” the group “have lys” turn off. Ahhh…
